How-to-Graph-Your-Network.md

```diff
@@ -16,7 +16,32 @@ To manage devices within Cacti, click on the Devices menu item. Clicking Add wil
 
 The field definitions are as follows
 
-||
+###### Table 7-1. Device Field Definitions
```

Comment:
You are getting this flagged because there are `============` lines above.  All headings should be converted to the `#` format.  Line 1 should have a single hash, Line 2 should be removed, Line 8 should have 2 hashes, line 9 removed. etc.